### PR TITLE
Emit event when receiving `Pong`

### PR DESCRIPTION
**Pull request by DerEwige**
create pong event and publish pongs, so plugins can subscribe to the latency of the peers

### CODE REVIEW

File: eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala [hunk 1]
```diff
@@ -31,4 +31,6 @@ case class PeerDisconnected(peer: ActorRef, nodeId: PublicKey) extends PeerEvent
 
 case class LastChannelClosed(peer: ActorRef, nodeId: PublicKey) extends PeerEvent
 
+case class receivedPong(latency: FiniteDuration, nodeId: PublicKey) extends PeerEvent
```
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala:34]: nit: to better match the rest of the codebase:

```suggestion
case class PongReceived(nodeId: PublicKey, latency: FiniteDuration) extends PeerEvent
```

**Review by t-bast — COMMENTED** [review]
Concept ACK, just some cosmetic changes needed.

File: eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala (hunk 1 above)
Comment (DerEwige) [reply eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala:34]: if you change the order in the event, you need to change the publush line too in PeerConnection.scala

Comment (t-bast) [reply eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala:34]: Yes, that's what I expect you to do when fixing this comment :)

Comment (t-bast) [reply eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala:34]: Also, you haven't properly imported `FiniteDuration`, the build is failing, that should be fixed.

**Review by t-bast — APPROVED** [review]

**Pull request merged**